.github/workflows/linux.yml

```diff
@@ -297,7 +297,7 @@ jobs:
           - goarch: arm
             toolchain: arm-linux-gnueabihf
             libc: armhf
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
I specifically picked an older version for wider compatibility. I'm not sure TinyGo binaries built on ubuntu-latest will run on ubuntu-22.04 for example (or any random old Raspberry Pi install).

Reply:
Hmm, that is possible. Should we leave it how it is now, since I have merged it, or should we add a new PR that downgrades?

Reply:
See: https://github.com/tinygo-org/tinygo/pull/4753